### PR TITLE
Fix ansible-lint warning: trailing spaces

### PR DESCRIPTION
**Pull request by yanhuih2022**

### CODE REVIEW

**Comment by Tomorrow9** [comment]
`git.diff` file you can remove it.

File: windows/utils/win_get_vmtools_driver_list.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@
     - win_powershell_cmd_output.stdout_lines is defined
     - win_powershell_cmd_output.stdout_lines | length != 0
 - ansible.builtin.debug:
-    msg: "Get VMware driver list: {{ vmtools_driver_list }}"
+  msg: "Get VMware driver list: {{ vmtools_driver_list }}"
```
Comment (keirazhang) [inline windows/utils/win_get_vmtools_driver_list.yml:22]: Please correct the indent of this msg line.

**Comment by keirazhang** [comment]
Please remove this file `git.diff`, it is not the project file. And after your change, please run test your changes with 1 linux OS and 1 windows OS. 

Thanks~

File: windows/memory_hot_add_basic/memory_hot_add_basic.yml [hunk 2]
```diff
@@ -14,9 +14,9 @@
   tasks:
     - block:
         - include_tasks: ../setup/test_setup.yml
- 
-        - block: 
-            # Refer to KB article https://kb.vmware.com/s/article/52584
+
+        - block:
+          # Refer to KB article https://kb.vmware.com/s/article/52584
```
Comment (keirazhang) [inline windows/memory_hot_add_basic/memory_hot_add_basic.yml:19]: Please keep the indent align with the following line.

File: common/vm_create.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@
 #   guest_id (required): the guest ID of new created VM.
 #   datastore (required): the name of datastore where new VM created.
 #   vm_power_state (optional): the power state of VM after creation, valid values are
-#     'poweredon', 'powered-on', 'poweredoff', 'powered-off', default value is 'poweredoff'. 
+#   'poweredon', 'powered-on', 'poweredoff', 'powered-off', default value is 'poweredoff'.
```
Comment (keirazhang) [inline common/vm_create.yml:10]: Please just remove the tailing space here. Leave the space at the beginning.

File: common/vm_create.yml [hunk 4]
```diff
@@ -18,7 +18,7 @@
 #   vm_cdroms (optional): VM CDROM list, this parameter is optional, but if you want to install
 #     guest OS after VM created, please configure the OS installation ISO files.
 #   boot_disk_controller (optional): valid values are 'buslogic', 'lsilogic', 'lsilogicsas',
-#     'paravirtual', 'sata' and 'nvme', default value is 'paravirtual'.
+#   'paravirtual', 'sata' and 'nvme', default value is 'paravirtual'.
```
Comment (keirazhang) [inline common/vm_create.yml:21]: Please just remove the tailing space here. Leave the space at the beginning.

**Comment by keirazhang** [comment]
Please also resolve conflicting files listed above

**Comment by yanhuih2022** [comment]
Qi, it looks conflicts have been resolved.
And for another task of unamed task, I modified under the same branch. So, it commits under the same pull. May I just leave the modification in this branch and will pull a new branch for the rest tasks? I saw there're still some places related to unamed task today.

If you think it will cost more time to close this pull. I will think other way to pick out the unamed task part.

Thanks for your review!

**Review by keirazhang — APPROVED** [review]
Look good now. Thanks!

**Pull request merged**